### PR TITLE
[RFR] fix scheme problem whith reverse proxy

### PR DESCRIPTION
**Pull request by zoneprog**
| Q | A
| --- | ---
| Branch? |  2.0
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Tests pass? | yes
| Fixed tickets | 727
| License | MIT
| Doc PR | 

Solution if you're behind a reverse proxy handling the HTTPS stuff and dispatching the requests as HTTP to various Docker containers running the actual Symfony application.

### CODE REVIEW

**Comment by daFish** [comment]
This looks like a fix which is also covered (in a different way) by #1059.

**Comment by maximgubar** [comment]
Hi @zoneprog ,
Thank you for the merge request,
could you please merge latest master in your branch in order to pass the CI ?

File: Imagine/Cache/Resolver/WebPathResolver.php [hunk 1]
```diff
@@ -143,25 +143,14 @@ protected function getFileUrl($path, $filter)
...
         if ('.php' === mb_substr($this->requestContext->getBaseUrl(), -4)) {
             $baseUrl = pathinfo($this->requestContext->getBaseurl(), PATHINFO_DIRNAME);
         }
         $baseUrl = rtrim($baseUrl, '/\\');
 
-        return sprintf('%s://%s%s%s',
-            $this->requestContext->getScheme(),
+        return sprintf('//%s%s',
             $this->requestContext->getHost(),
-            $port,
```
Comment (dbu) [inline Imagine/Cache/Resolver/WebPathResolver.php]: this will break if a site runs on a non-default port. i think we do need to keep the port handling.

**Comment by dbu** [comment]
see https://symfony.com/doc/current/deployment/proxies.html

**Pull request closed without merging**